bundles/org.openhab.ui.basic/src/main/java/org/openhab/ui/basic/internal/render/PageRenderer.java

```diff
@@ -136,12 +136,12 @@ public StringBuilder processPage(String id, String sitemap, String label, EList<
         return preChildren.append(postChildren);
     }
 
-    private void processChildren(StringBuilder sb_pre, StringBuilder sb_post, EList<Widget> children, String sitemap)
+    private void processChildren(StringBuilder sbPre, StringBuilder sbPost, EList<Widget> children, String sitemap)
             throws RenderException {
         // put a single frame around all children widgets, if there are no explicit frames
         if (!children.isEmpty()) {
-            EObject firstChild = children.get(0);
-            EObject parent = itemUIRegistry.getParent((Widget) firstChild);
+            Widget firstChild = children.get(0);
+            EObject parent = itemUIRegistry.getParent(firstChild);
             if (!(firstChild instanceof Frame || parent instanceof Frame || parent instanceof Sitemap)) {
```

Comment:
We have to be sure that this change of type does not break the test `firstChild instanceof Frame`

Reply:
Frame => LinkableWidget => Widget => EObject
So it is OK.